includes/Modules/Analytics.php

```diff
@@ -939,11 +939,21 @@ protected function create_analytics_site_data_request( array $args = array() ) {
 
 		$dimension_filter_clauses = array();
 
-		$hostname         = wp_parse_url( $this->context->get_reference_site_url(), PHP_URL_HOST );
+		$hostnames = array_values(
+			array_unique(
+				array_map(
+					function ( $site_url ) {
+						return wp_parse_url( $site_url, PHP_URL_HOST );
+					},
+					$this->permute_site_url( $this->context->get_reference_site_url() )
+				)
+			)
+		);
```

Comment:
One last thing here that I missed before – the added values + unique calls aren't necessary since the values will always be unique (one will always be the current hostname, and the other a www-added variant).
```suggestion
		$hostnames = array_map(
			function ( $site_url ) {
				return wp_parse_url( $site_url, PHP_URL_HOST );
			},
			$this->permute_site_url( $this->context->get_reference_site_url() )
		);
```